JavaScript/JavaScriptSDK.Tests/Selenium/Tests.cs

```diff
@@ -67,9 +67,40 @@ public void Firefox_CodeCoverage()
             ffProfile.SetPreference("browser.download.folderList", 2);
             ffProfile.SetPreference("browser.helperApps.neverAsk.saveToDisk", "text/plain");
 
-            RunTest(new OpenQA.Selenium.Firefox.FirefoxDriver(ffProfile), true);
+            RunTest(new OpenQA.Selenium.Firefox.FirefoxDriver(ffProfile), PATH_TO_TESTS, true);
         }
 
+        [TestMethod]
+        public void Firefox_E2E_DisableTelemetryTests()
+        {
+            RunTest(new OpenQA.Selenium.Firefox.FirefoxDriver(), "/E2ETests/E2E.DisableTelemetryTests.htm");
+        }
+
+        [TestMethod]
+        public void Firefox_E2E_PublicApiTests()
+        {
+            RunTest(new OpenQA.Selenium.Firefox.FirefoxDriver(), "/E2ETests/E2E.PublicApiTests.htm");
+        }
+
+        [TestMethod]
+        public void Firefox_E2E_SanitizerE2ETests()
+        {
+            RunTest(new OpenQA.Selenium.Firefox.FirefoxDriver(), "/E2ETests/E2E.SanitizerE2ETests.htm");
+        }
+
+        // Tests are failing, need to fix before we enable them.
+        /* [TestMethod]
+        public void Firefox_E2E_autoCollection()
```

Comment:
These two tests are failing and I'm fixing them right now. I'll enable them in a separate pull request.